deliver/spec/upload_screenshots_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'spec_helper'
+require 'fakefs/spec_helpers'
+require 'stubs/upload_stubbing'
+
+describe Deliver::UploadScreenshots do
+  include FakeFS::SpecHelpers
+
+  before :each do
+    @root = '/tmp/screenshots/'
+    FileUtils.mkdir_p(@root)
+
+    (1..5).each do |i|
+      file_path = File.join(@root, "scr_#{i}.jpg")
+      File.open(file_path, 'w') { |file| file.write("#{i}") }
+    end
+  end
+
+  def mock_local_upload(sequence)
+    @order = 0
+    to_upload = sequence.map do |i|
+      file_path = File.join(@root, "scr_#{i}.jpg")
+      md5 = Spaceship::Utilities.md5digest(file_path)
+      file = Screenshot.new({file_path: file_path, language: 'en-US', original_file_name: "ftl_#{md5}_scr_#{i}.jpg", sort_order: @order})
```

Comment:
Apple ITCScreenshot starts form 1. Local upload starts normally form 0, so it's closer to real life